io/disk/Avago_storage_adapter/avago3008.py

```diff
@@ -44,9 +45,13 @@ def setUp(self):
         self.disk = str(self.params.get('disk')).split(" ")
         self.spare = str(self.params.get('spare'))
         self.size = int(self.params.get('size', default='max'))
+        self.tool_location = self.params.get('tool_location')
         if not self.disk:
             self.skip("Please provide disks to run the tests")
         self.number_of_disk = len(self.disk)
+        if (not os.path.isfile(self.tool_location) or not
```

Comment:
@vrbagalkote we can have this check immediately after reading the yaml parameter `self.tool_location`. Except that, looks good.